src/Proto.Cluster/PidCache.cs

```diff
@@ -53,7 +55,7 @@ public Task ReceiveAsync(IContext context)
             switch (context.Message)
             {
                 case Started _:
-                    //  Console.WriteLine("Started PidCacheActor");
+                    _logger.LogDebug("Started PidCacheActor");

```

Comment:
This one is used behind a router, so we get 100 of these, which might be a bit too much :-)